integr8ly/integreatly-operator
# MGDAPI-5266 - obervability: v4.1.2

# Issue link

https://issues.redhat.com/browse/MGDAPI-5266

# What


# Verification steps



* Checkout master and install RHOAM
```
export LOCAL=false
make cluster/prepare/local
make code/run
```
* Wait until install completes successfully
* Stop the operator
* Checkout this branch 
* Re-run operator
```
make code/rerun
```
* Verify observability operator is upgraded
  *  NOTE: It will step through the upgrade to 4.1.1 before upgrading to 4.1.2
```
watch "oc get csv -n redhat-rhoam-observability-operator; oc get pods -n redhat-rhoam-observability-operator; oc get pods -n redhat-rhoam-observability; echo ""; echo "RHMI CR"; oc get rhmi rhoam -n redhat-rhoam-operator -o jsonpath='{.status.stage}'" 
```

* Verify the prometheus index image is `quay.io/integreatly/custom-prometheus-index:1.0.1`
```
oc get pods -l olm.catalogSource=prometheus-catalogsource -n redhat-rhoam-observability -o json | jq '.items[0].spec.containers[0].image'
```
